API.md

```diff
@@ -0,0 +1,203 @@
+# CodeReady Analytics Core API Specifications
+
+CodeReady Analytics is an open-source analytics platform aimed primarily at assisting developers in getting insights and recommendations on the selected dependency/package stack used for developing their applications.
+
+This documentation is for developers who are interested in integrating their services or applications with CodeReady Analytics platform.
+
+## **Supported ecosystems**
+
+Below are the ecosystems that CodeReady Analytics currently supports.  We will be adding support for more ecosystems in the near future.
+| | |
+|-|-|
+|**Ecosystem**|**Manifest file**|
+| Java Maven | pom.xml |
+| Python Pypi | requirements.txt |
+| NodeJS | package.json |
+
+## **Prerequisites**
+
+Any new integration with the CodeReady Analytics platform will have to do that using an existing 3-scale API management gateway setup. Please reach out to us by raising an issue [here](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/issues)
+
+## **Initiate Dependency Analysis**
+
+Submits a dependency stack analysis request
+
+- **URL**
+  `https://<3scale-api-gateway-url>/api/v1/stack-analyses`
+
+- **Method**
+  `POST`
+
+- **Response Format**
+  `jSON`
+
+- **Path Parameters**
+  `None`
+
+- **Query Parameters**
+  | | | | |
```

Comment:
There is a problem with rendering tables, see the rendered version: https://github.com/fabric8-analytics/fabric8-analytics-server/blob/0cd38a536bea35a6e3d08e212f3e86ee11b60143/API.md 
Ideallu I would suggest you use Asccidoc format  for the docs but if you want to use mark-down, see here for information on tables in markdown: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables